git-cliff-core/src/commit.rs

```diff
@@ -423,8 +433,9 @@ impl Serialize for Commit<'_> {
 			}
 		}
 
-		let mut commit = serializer.serialize_struct("Commit", 9)?;
+		let mut commit = serializer.serialize_struct("Commit", 20)?;
```

Comment:
I counted there are 20 fields, so I put these as 20. Not sure if it is correct.
I found in `cargo` they use separated struct for serializing. Maybe we should do that as well?
https://github.com/rust-lang/cargo/blob/82c489f1c612096c2dffc48a4091d21af4b8e046/src/cargo/core/manifest.rs#L324

Reply:
Yup, 20 should be correct.

>  they use separated struct for serializing.

I'm not sure. If you think it will simplify the code, yes. But the current serialization is already super straightforward :)

What would be the benefit of using a separate struct?

Reply:
> What would be the benefit of using a separate struct?

Actually, I'm pretty new to serde so I'm not quite sure.
But, I think we don't need to manually change the number of fields when using separated struct.
We can also use proc macro directly, which is more friendly.

But those fields aren't change quite often, so I think just leave them there is also good.